source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -0,0 +1,402 @@
+---
+title: External Network Providers
+category: feature
+authors: mmirecki
+wiki_category: Feature
+wiki_title: Features/External Network Providers
+wiki_revision_count: 1
+wiki_last_updated: 2016-01-25
+feature_name: External Network Providers
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# External Network Providers
+
+
+## Summary
+
+Currently the Openstack Neutron integration allows oVirt to integrate with Openstack Neutron. This purpose of this feature is to simplify this to make it easier to add support for networks supplied by external providers. The providers will be able to provide networks using a simplified Neutron-like API.
+
+
+### Owner
+
+*   Feature Owner: Marcin Mirecki: [ mmirecki](User:mmirecki)
+
+<!-- -->
+
+*   Email: <mmirecki@redhat.com>
+
+### Benefit to oVirt
+
+oVirt will be able to use networks definded by external providers.
+oVirt will provide an API for the external providers allowing to easily integrate with oVirt.
+
+
+## Detailed Description
+
+Many organizations use centralized network management systems to handle all their networking, and would like them to manage the network topology of their oVirt environments. Currently the only option of using external networks in oVirt is to used the OpenStack Neutron integration, which allows the import of OpenStack Neutron networks and the provisioning of VM's connected to these networks. This feature is however very specific to OpenStack Neutron. We would like to extend this API to other external network providers, by making it simpler, more general and less dependant on OpenStack Neutron features. The result should be an API over which oVirt can communicate with external network management systems, and use the networks defined in them in provisioned VMs.
```

Comment:
We would like to extend this *functionality*  to other external (we do not intend to extend any API)

BTW, it seems that shorter lines allow easier review.